array-methods/filter.js

```diff
@@ -0,0 +1,19 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna',
+];
+
+const evens = numbers.filter((a) => a % 2 === 0);
+console.log('evens', evens);
+
+const noD = names.filter((a) => a.includes('d', 'D'));
```

Comment:
This doesn't work. `includes` only accepts on parameter to match against. It should also match `'Donna'` but doesn't.

Finally, you are logging the names that _do_ contain a 'd' instead of those that _don't_.